modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/chart/__snapshots__/TooltipContent.es.js.snap

```diff
@@ -0,0 +1,112 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Tooltip renders tooltip for MultiBar Chart 1`] = `
```

Comment:
@alinedoleron, do we need to test it using snapshots or can we test it in other way?